libs/components/src/lib/tabs/tabs.ts

```diff
@@ -0,0 +1,44 @@
+import { observable } from '@microsoft/fast-element';
+import { Tabs as FoundationTabs } from '@microsoft/fast-foundation';
+
+const TABS_ACTIVE_INDICATOR_INLINE_SIZE = '--_tabs-active-indicator-inline-size';
+
+/**
+ * Base class for tabs
+ *
+ * @public
+ */
+export class Tabs extends FoundationTabs {
+	@observable tablist?: HTMLElement;
+
+	override orientationChanged(): void {
+		super.orientationChanged();
+		this.patchIndicatorStyleTransition();
+		if (this.orientation === 'vertical') {
```

Comment:
```suggestion
		if (this.orientation === TabsOrientation.vertical) {
```
(requires adding `TabsOrientation` to the imports)